scss/skin/_default.scss

```diff
@@ -56,6 +54,11 @@
   }
 }
 
+.oo-video-player {
+  // default aspect ratio 16:9
+  @include set-aspect-ratio(16, 9);
```

Comment:
Would setting this on a different class have any effect on the auto aspect ratio feature of the player?

Reply:
I pulled this out from the .oo-player class into its own .oo-video-player class, which is not applied for the audio only player (since we do not want auto aspect ratios for the audio player), but is applied to the same div that .oo-player is for the video player. Original location here:

https://github.com/ooyala/html5-skin/blob/master/scss/skin/_default.scss#L18

This should not have any effect for the auto aspect ratio feature for the video player.

Reply:
Oh ok, got it. So this `oo-video-player` class is set on the same container as `oo-player` right?

Reply:
Yep